src/index.html

```diff
@@ -22,6 +22,431 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a
+          href="index.html"
+          class="logo"
+        >
+          <img
+            src="./images/logo.png"
```

Comment:
The `alt` attribute for the logo image is currently set to 'Moyo logo'. Consider providing a more descriptive text that conveys the purpose of the image, especially if it's used as a link.